ss/ssm/ssm.go

```diff
@@ -19,9 +15,3 @@ func New(h string, a rsapi.Authenticator) *API {
 	ssAPI := API{API: api}
 	return &ssAPI
 }
-
-// BuildHTTPRequest wraps the underlying rsapi implementation and simply prefixes the path with
-// the service manager path.
-func (a *API) BuildHTTPRequest(verb, path, version string, params, payload rsapi.APIParams) (*http.Request, error) {
-	return a.API.BuildHTTPRequest(verb, "/manager"+path, version, params, payload)
-}
```

Comment:
Yes, because now the API JSON explicitly includes "/api/<service>" whereas before the SS paths were simply, "/collections/60073/applications" etc.